### ISSUE TITLE
Remove extra `<h1>` on Personal Details Form

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5062]
In the `/sign-up` flow, there are two `<h1>`s being rendered on the following page `/sign-up/identity-verification`: "Sign up for SimpleReport" and "Personal details".

![Screen Shot 2023-01-19 at 11.49.43.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/b0c649a3-33c8-487b-805d-5baa2870f06e)

**Action requested**
Remove the unnecessary `<h1>`

**Comment by emmastephenson** [comment]
Personal details should be an <h2>
